apps/ui/src/helpers/utils.ts

```diff
@@ -475,18 +475,24 @@ export function getChoiceWeight(
   return isNaN(percent) ? 0 : percent;
 }
 
-export function getChoiceText(
-  availableChoices: string[],
-  choice: number | number[] | Record<string, number>
-) {
+export function getChoiceText(availableChoices: string[], choice: Choice) {
```

Comment:
Not sure why those changes were introduced in this PR?

Reply:
We use `getChoiceText()` to preview the user's choice in the vote modal.

The previous function does not return anything when the choice is invalid or blank (in the case of weighted choice)